common/common-rest/src/main/java/io/servicecomb/common/rest/codec/param/RestClientRequestImpl.java

```diff
@@ -69,6 +83,34 @@ public void end() throws Exception {
     request.end(bodyBuffer);
   }
 
+  private void attachFiles() {
+    if (!uploads.isEmpty()) {
```

Comment:
if mixed upload and normal form parameter, then all form parameter(include upload) must encode as multipart form data
so there is conflict logic between attachFiles and genBodyBuffer